src/Analysis/Ast/Impl/Indexing/IndexManager.cs

```diff
@@ -35,6 +35,8 @@ internal class IndexManager : IIndexManager {
         private readonly CancellationTokenSource _allIndexCts = new CancellationTokenSource();
         private readonly TaskCompletionSource<bool> _addRootTcs;
 
+        public const int DefaultWorkspaceSymbolsLimit = 50;
```

Comment:
The old LS limit was 1000. https://github.com/Microsoft/python-language-server/blob/7f5729a40cf3a994f013a9a4812a71e5a24eba03/src/LanguageServer/Impl/Implementation/Server.WorkspaceSymbols.cs#L33

Reply:
Changed it. Now the interface requires an explicit limit

Reply:
@MikhailArkhipov 	We decided to just require a limit, since I can't think of a reason you'd want to allow it to collect a list unbounded. If we want full enumerable access, we can probably modify the interface later, but this works fine for the LS (which applies a limit anyway).

Reply:
Sure.